composer.json

```diff
@@ -208,8 +208,19 @@
         "allow-plugins": true
     },
     "scripts": {
+        "build-theme": [
+            "npm ci --prefix docroot/themes/humsci/humsci_basic",
+            "npm run build --prefix docroot/themes/humsci/humsci_basic"
+        ],
+        "test-theme": [
+            "npm ci --prefix docroot/themes/humsci/humsci_basic",
+            "npm run test --prefix docroot/themes/humsci/humsci_basic"
+        ],
         "post-drupal-scaffold-cmd": [
             "patch -p1 < patches/scaffold.patch"
+        ],
+        "post-install-cmd": [
+            "@build-theme"
```

Comment:
What's the purpose of compiling theme assets on `composer install`? It's a bit of a hindrance on backend dev workflows.

Reply:
Our goal here was to have a fully operational site with all assets build after the install, but you're right, it can impact BE work. I'll remove it from there.

Reply:
There's a separate step defined in `blt.yml` that builds the theme.